src/containers/simpleReview/SimpleReviewPage.ts

```diff
@@ -0,0 +1,37 @@
+import type { AppState, ValidRecord } from "../../types";
+import type { Dispatch } from "redux";
+import type { StateProps } from "../../components/simpleReview/SimpleReview";
+
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import SimpleReview from "../../components/simpleReview/SimpleReview";
+import * as SignoffActions from "../../plugins/signoff/actions";
+import { getClient } from "../../client";
+
+function fetchRecords(bid: string, cid: string): Promise<ValidRecord[]> {
+  const coll = getClient().bucket(bid).collection(cid);
+  return coll.listRecords().then(({ data: records }) => {
```

Comment:
You can ask the server to sort the records if you prefer: https://github.com/Kinto/kinto-http.js/#sorting